content/docs/examples/multicluster/icp/index.md

```diff
@@ -0,0 +1,225 @@
+    PING 20.1.47.150 (20.1.47.150) 56(84) bytes of data.
+    64 bytes from 20.1.47.150: icmp_seq=1 ttl=63 time=0.759 ms
+    {{< /text >}}
+
+The above configuration completely configured full IP route mesh across all nodes in the two IBM Cloud Private Clusters,
+which enables Pod communication cross clusters.
+
+## Install the Istio for multi cluster
+
+[Follow the multicluster installation steps](/docs/setup/kubernetes/multicluster-install/) to install and config
+local Istio control plane and Istio remote on `cluster-1` and `cluster-2`.
+
+This example uses `cluster-1` as local Istio control plane and `cluster-2` as Istio remote.
+
+## Deploy Bookinfo Example Across Clusters
+
+__NOTE__: The following example enables [sidecar automatic injection](/docs/setup/kubernetes/sidecar-injection/#automatic-sidecar-injection).
+
+1.  Install bookinfo on the `cluster-1`. Remove the `reviews-v3` deployment to deploy on remote:
+
+    {{< text bash >}}
+    $ kubectl apply -f samples/bookinfo/platform/kube/bookinfo.yaml
+    $ kubectl apply -f samples/bookinfo/networking/bookinfo-gateway.yaml
+    $ kubectl delete deployment reviews-v3
+    {{< /text >}}
+
+1.  Create the `reviews-v3.yaml` manifest for deployment on the remote:
+
+    {{< text yaml plain "reviews-v3.yaml" >}}
+    ---
+    ##################################################################################################
+    # Ratings service
+    ##################################################################################################
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: ratings
+      labels:
+        app: ratings
+    spec:
+      ports:
+      - port: 9080
+        name: http
+    ---
+    ##################################################################################################
+    # Reviews service
+    ##################################################################################################
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: reviews
+      labels:
+        app: reviews
+    spec:
+      ports:
+      - port: 9080
+        name: http
+      selector:
+        app: reviews
+    ---
+    apiVersion: extensions/v1beta1
+    kind: Deployment
+    metadata:
+      name: reviews-v3
+    spec:
+      replicas: 1
+      template:
+        metadata:
+          labels:
+            app: reviews
+            version: v3
+        spec:
+          containers:
+          - name: reviews
+            image: istio/examples-bookinfo-reviews-v3:1.5.0
+            imagePullPolicy: IfNotPresent
+            ports:
+            - containerPort: 9080
+    EOF
+    {{< /text >}}
+
+    _Note:_ The `ratings` service definition is added to the remote cluster because `reviews-v3` is a
+    client of `ratings` and creating the service object creates a DNS entry.  The Istio sidecar in the
+    `reviews-v3` pod will determine the proper `ratings` endpoint after the DNS lookup is resolved to a
```

Comment:
I think we need a "how multicluster works blog" :)

Reply:
@sdake do you mean this https://github.com/istio/istio.github.io/issues/1381 ? Add a detailed use case for multicluster?

Reply:
Yup, that is a separate task - thanks i forgot that was in the issue queue.